modules/custom_operations/user_ie_extensions/tokenizer/CMakeLists.txt

```diff
@@ -168,8 +166,12 @@ set_target_properties(${TARGET_NAME} PROPERTIES COMPILE_OPTIONS "${extra_flags}"
 # Post build steps to copy core_tokenizers dependencies
 #
 
+set_property(DIRECTORY ${sentencepiece_SOURCE_DIR} PROPERTY EXCLUDE_FROM_ALL ON)
+
 if(BUILD_FAST_TOKENIZERS)
-  # TODO
+  install(TARGETS core_tokenizers
+    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
```

Comment:
can we just set `set(extra_libs $<TARGET_FILE:core_tokenizers>)` ? and keep the of file unchanged

Reply:
it doesn't work